pkg/cli/cmd/uninstall/kubernetes/kubernetes.go

```diff
@@ -91,13 +92,21 @@ func (r *Runner) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	if !state.Installed {
+	if !state.RadiusInstalled {
 		r.Output.LogInfo("Radius is not installed on the Kubernetes cluster")
 		return nil
 	}
 
 	r.Output.LogInfo("Uninstalling Radius...")
-	err = r.Helm.UninstallRadius(ctx, r.KubeContext)
+	err = r.Helm.UninstallRadius(ctx, helm.ClusterOptions{
+		Radius: helm.ChartOptions{
+			Namespace: helm.RadiusSystemNamespace,
+		},
+		Dapr: helm.ChartOptions{
```

Comment:
Is this only going to delete Dapr if it is installed via Radius? What if I have a Dapr installation on my cluster that I installed manually? Is that also going to be deleted if I run `rad uninstall kubernetes`?

Reply:
no it won't be deleted, though my feelings are that we should clean up after ourselves, but it gets complicated during upgrades etc

Reply:
> What if I have a Dapr installation on my cluster that I installed manually?

The problem is exactly this. We won't later whether or not we're the ones that installed Dapr. 

We should do the safe thing.

Reply:
Following up on this discussion, we investigated and it turns out Radius also uninstall Contour when removed. 

Since both decisions have downsides, we're going to be consistent and also uninstall Dapr until we get user feedback to the contrary.